my_list.rb

```diff
@@ -0,0 +1,11 @@
+# require './my_list.rb'
```

Comment:
- [ ] Please kindly remove this commented line of code and import the appropriate module needed for the execution of your app. ✔️ At the moment, your app did not run as expected majorly because of the reason stated above. ℹ️ 

```suggestion
# require './my_enumerable.rb'
```